rnaseq/tasks/main.yml

```diff
@@ -126,15 +126,15 @@
 - name: Create NFS directory
   become: yes
   file: path=/nfs/data state=directory mode=0755
-- name: mount /nfs/data
-  become: yes
-  mount: fstype=nfs opts='vers=3' path=/nfs/data src={{ readonly }} state=mounted
+#- name: mount /nfs/data
+#  become: yes
+#  mount: fstype=nfs opts='vers=3' path=/nfs/data src={{ readonly }} state=mounted
```

Comment:
Is this commented out code left in for a reason?

Reply:
I removed this yesterday but forgot to push the commit 🤦‍♂